doc/src/conf.py

```diff
@@ -31,8 +31,8 @@
 # To stop docstrings inheritance.
 autodoc_inherit_docstrings = False
 
-# MathJax file, which is free to use.  See http://www.mathjax.org/docs/2.0/start.html
-mathjax_path = 'http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML-full'
+# MathJax file, which is free to use.  See https://www.mathjax.org/#gettingstarted
+mathjax_path = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_HTML-full'
```

Comment:
Is it possible to use a "latest" URL instead of a specific version here?

Reply:
Not with cdnjs, which is the CDN that [MathJax people recommend](http://docs.mathjax.org/en/latest/start.html#mathjax-cdn). 
But there are alternative CDNs on their list that provide rolling release, serving code from npm.  So we can use 
https://cdn.jsdelivr.net/npm/mathjax/unpacked/MathJax.js or 
https://unpkg.com/mathjax/unpacked/MathJax.js
which redirect to the latest version. Is this preferable?

Reply:
If we pin 2.7.2 it will likely never get updated, because nobody looks at this. I'm not so sure about the alternative cdns, though. How stable are they? What is the risk of them going away at some point in time.

Reply:
If nobody notices that MathJax is old, is there a reason to change the version? Of all the [formatting problems](http://docs.sympy.org/dev/modules/polys/ringseries.html#sympy.polys.ring_series.rs_series_from_list) that docs have, MathJax version seems one of least. 

I don't know anything about those CDNs. It's possible that some of the alternative CDNs will go away before we will have a reason to change the version of MathJax. The current major version of MathJax, 2.7, was released in 2016.